bin-helper.js

```diff
@@ -41,6 +41,11 @@ export default function binHelper(options) {
         extensions.push(ext);
     });
 
+    // --no-timeouts option
+    if (typeof options.timeouts === 'boolean') {
```

Comment:
That's probably the only thing which seems weird for me: is it mocha which translates --no-timeouts to "timeouts"?

Reply:
I think it is done by yargs https://github.com/yargs/yargs/blob/master/docs/tricks.md#negate

Reply:
Gotcha